### ISSUE TITLE
Edit of Ingredient Description Will Not Save

### ISSUE THREAD

**Issue by zombiewoof** [kirienko/gourmet#249]
Edit ingredient description will not be saved.

Choose a recipe for edit. Change the text description of an ingredient. Save will become clickable, but nothing happens when it is clicked - view of recipe card shows original description.

Possibly related: Change in Notes (made after above test) is not saved when save is clicked. Save still clickable from above test.

Closing edit window, after both tests were performed resulted in warning about unsaved data. Clicking save in the warning box did not save data.

### Environment
gourmet-5ad618be
Operating System: Ubuntu 20.04 (with latest updates)
Desktop: Cinnamon 4.4.8
Installed as Flatpak

**Comment by cydanil** [comment]
Thanks, I was able to reproduce the issue with the ingredients editing. A fix is coming

Regarding the second issue, I was able to reproduce it in the version you use with slightly different results:  
- Editing the `Notes` did prompt me to save the changes, which I did, but the recipe card did not update;  
- Closing and re-opening the recipe card showed the updated `Notes`.

I tested this both when adding and removing text.

If you observe the same behaviour, then it's been solved already.

**Comment by zombiewoof** [comment]
Regarding the second issue. In the original test I did not close and re-open the card view.

Just now, I opened the application and viewed the recipe card, that I changed earlier, and the change was, indeed, there. I made another change and clicking Save immediately updated the card view. This test was done without changing anything in ingredients.

Maybe the problem had to do with editing Notes AFTER I'd edited an ingredient description. The Save icon never greyed out, as it should after saving the ingredient description, in my original test. I'm thinking that the delay in card Notes update is related to the failure to save updated ingredient description.
